src/AVFoundation/AVCaptureDeviceDiscoverySession.cs

```diff
@@ -19,7 +21,7 @@ public static AVCaptureDeviceDiscoverySession Create (AVCaptureDeviceType [] dev
 		{
 			var arr = new NSMutableArray ();
 			foreach (var device in deviceTypes)
-				arr.Add (device.GetConstant ());
+				arr.Add (device.GetConstant ()!);
```

Comment:
If the contestant is null? Shall we just not add it? What is the reasoning behind the use of "!"

Reply:
I just tried to avoid introducing any new changes, but I don't see an issue with skipping if it is null!